### PR TITLE
Mobile view

### PR DESCRIPTION
**Pull request by musyelevateher**
Added "Html and CSS" to the webpage

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+margin-left: 400px;
+display: inline-flex;
+gap: 30px;
+font-family: "urbanist";
+font-weight: 600;
+font-size: 16px;
+line-height: 24px;
+color: #595959;
+padding-top: 30px;
+}
```
Comment (Devsalmasaeed) [inline style.css]: Hi @musyelevateher,

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    Remote Collaboration
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️
- Make your HTML truly semantic, ensure that the HTML tags correctly describe the content's meaning and structure.
- Please use professional PR title
- Make your design UI like the Figma design

# Figma design

image


image

# Your design


image


image




Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

**Review by Devsalmasaeed — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+              <div class="lake">
+                              <img class="rotate" src="asset/sunter lake.svg" alt="">
+                              <h4 class="name">Sunter Lake</h4>
+              <div class="location">
+                                <img src="asset/Location.svg" alt="">
+                                <h3 class="country">Jawa Tengah, Indonesia</h3>
+              <div class="lake">
+                                  <img class="rotate" src="asset/negeri.svg" alt="">
+                                  <h5 class="name">Negeri di atas Awan</h5>
+              <div class="location">
```
Comment (Xfarouq) [inline index.html]: Why are your closing div tags arranged like this? This method isn't applicable for best practice, please make sure to rectify it

**Review by Xfarouq — CHANGES_REQUESTED** [review]
Hi @musyelevateher 

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    Remote Collaboration
</div>

### To Highlight 💯
- **Good use of Gitflow** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️
- **Make sure to have a clean code always**
- Please make sure to follow exactly how the mobile view was implemented in the tutorial video 🙏
- The mobile view is okay , but don't specify a width to it except to use media queries for that


### Optional Suggestions
**I strongly recommend you to watch the tutorial videos very well please for the mobile first approach

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

File: style.css [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+ul {
+list-style: none;
+display: flex;
+gap: 30px;
+margin-left: -30px;
+}
```
Comment (Devsalmasaeed) [inline style.css:332]: Hi @musyelevateher,

Your project is complete! There is nothing else to say other than... it's time to merge it!!!
Congratulations! 🎉

<div style="text-align:center;">
    Remote Collaboration
</div>

### Highlights
- ✅ **Good use of Gitflow**
- ✅ **UI looks like the Figma design**
- ✅ **Good use of HTML semantics**
- ✅ **Professional commit messages**
- ✅ **Professional PR title**
- ✅ **Professional PR description**

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

**Review by Devsalmasaeed — APPROVED** [review]

**Pull request merged**